playwright/test/search-images.test.ts

```diff
@@ -28,41 +25,22 @@ test('(1) | Search by term, filter by colour, check results, view image details,
   await newSearch(context, page, 'images');
   await searchQuerySubmitAndWait('art of science', page);
 
-  if (isMobile(page)) {
-    await page.locator('button[aria-controls="mobile-filters-modal"]').click();
-    await expect(await page.locator('#mobile-filters-modal')).toBeVisible();
-    await page.getByTestId('swatch-red').click();
-    await page.getByRole('button', { name: 'Close modal window' }).click();
-  } else {
-    await page.locator('button[aria-controls="images.color"]').click();
-    await page.getByRole('button', { name: 'Red', exact: true }).click();
-    await page.locator('button[aria-controls="images.color"]').click();
-  }
-  await expect(
-    await page.getByTestId('image-search-results-container')
-  ).toBeVisible();
+  await selectAndWaitForColourFilter(page);
   await expect(
-    await page
-      .locator('[data-testid="image-search-results-container"] li')
-      .first()
```

Comment:
To confirm my understanding; this was removed because (I think), it was looking at the first `li` to make sure there was at least one result. But `page.getByTestId('image-search-results-container')` would fail if there were no results anyway?

Reply:
`await clickActionClickSearchResultItem(1, page);` clicks on the 1st result. I assume this would fail if there were no result, but either way the image pop up would not open so `await expect(page.getByLabel('View expanded image')).toBeVisible();` would fail

Reply:
I think `page.getByTestId('image-search-results-container')` would also fail if no result btw, as the container only renders for image results

Reply:
agreed. works for me! Was a faffy one as well.